paseos/paseos.py

```diff
@@ -36,14 +46,19 @@ def __new__(self, local_actor: BaseActor):
             )
         return self.instance
 
-    def __init__(self, local_actor: BaseActor):
+    def __init__(self, local_actor: BaseActor, cfg=None):
         logger.trace("Initializing PASEOS")
-        self._cfg = load_default_cfg()
+        if cfg is not None:
+            self._cfg = cfg
+        else:
+            self._cfg = load_default_cfg()
```

Comment:
```suggestion
            self._cfg = cfg
```
Already taken care of in init I think?